src/app/models/Tag.ts

```diff
@@ -26,36 +26,28 @@ export interface TagInterface {
 /**
  * Tag model
  */
-export class Tag extends AbstractModel implements TagInterface {
+export class Tag extends AbstractModel implements ITag {
   public readonly kind: "AbstractModel" = "AbstractModel";
   public readonly id?: Id;
   public readonly text?: string;
   public readonly count?: number;
```

Comment:
this field should not be returned by the API?

Reply:
Its shown in the tags list view and i didnt want to create 25 api requests when i display the list of tags. If its not needed, we can remove it from the view: https://staging.ecosounds.org/admin/tags which also removes the issue

Reply:
leave it commented - it's not a field we'd unusually return since it is an aggregate value

Reply:
As in comment out the key, or leave a comment explaining that its in limbo? The value is already in use on the tag list page so commenting it out would require me to modify that too.

Reply:
Comment in limbo is fine.

This will have to be revisited because I do expect that field will  be output by the API. It contravenes the API conventions.

It's a little different for the rails view: it can do arbitrary queries anywhere anytime - which is what it does. _count_ is not part of the model server side either.